DirectEmployers/MyJobs
# [PD 2658] network template widgets

This is more than just editing the widgets and making them into individual pieces. There is also the design concept merged in along with the functionality of the left hand facets having functionality in them along with the design to them. Also, the job details, homepage, job results, etc pages are also in order with the design and formatting with widgets as well.

To Test:
In the admin page, make sure that you are set on home_page_listing_bootstrap3.html and then the url to go to should be my.jobs. This will take you to the homepage for testing around on.
